### PR TITLE
refresh after adding tracks to Playlist

### PR DESCRIPTION
**Pull request by qcasey**
This is fixed when calling 
`    await refresh();`
in the addTracks function, exactly like when calling rename() or remove()

### CODE REVIEW

**Comment by qcasey** [comment]
this is for #504 by the way

File: src/ui/actions/PlaylistsActions.ts [hunk 1]
```diff
@@ -123,6 +123,7 @@ export const addTracks = async (_id: string, tracksIds: string[], isShown?: bool
     const playlist = await app.models.Playlist.findOneAsync({ _id });
     const playlistTracks = playlist.tracks.concat(tracksIds);
     await app.models.Playlist.updateAsync({ _id }, { $set: { tracks: playlistTracks } });
+    await refresh();
```
Comment (martpie) [inline src/ui/actions/PlaylistsActions.ts]: I noticed this bug as well but I am not sure this is the "right" way to fix it, I'll need to investigate a tiny bit before accepting this.

**Comment by martpie** [comment]
By the way, I've fixed the checks on master, so if you merge/rebase, it should be green ;)

**Comment by martpie** [comment]
This is actually the correct solution, the data management is a bit messy I got to say, I should rework that a little at some point.

**Pull request merged**